test/TestHelper.cpp

```
@@ -526,12 +545,38 @@ bytes importData(json_spirit::mObject const& _o)
 	return data;
 }
 
+std::string compileLLL(std::string const& _code)
+{
+	FILE *fp;
+	char path[1035];
+	FILE *ft = fopen("temp.sol", "w");
+	if (ft == NULL)
+		cerr << "Error creating temp file for lllc";
```

We should throw an exception if anything fails